modules/javafx.base/src/main/java/com/sun/javafx/collections/ObservableListWrapper.java

```diff
@@ -25,27 +25,26 @@
 
 package com.sun.javafx.collections;
 
-import javafx.collections.ModifiableObservableListBase;
-import com.sun.javafx.collections.NonIterableChange.SimplePermutationChange;
-
 import java.util.BitSet;
 import java.util.Collection;
 import java.util.Comparator;
 import java.util.List;
 import java.util.RandomAccess;
+
+import com.sun.javafx.collections.NonIterableChange.SimplePermutationChange;
+
 import javafx.beans.InvalidationListener;
 import javafx.beans.Observable;
+import javafx.collections.ModifiableObservableListBase;
 import javafx.util.Callback;
 
 /**
  * A List wrapper class that implements observability.
  *
  */
-public class ObservableListWrapper<E> extends ModifiableObservableListBase<E> implements
-        SortableList<E>, RandomAccess {
+public class ObservableListWrapper<E> extends ModifiableObservableListBase<E> implements SortableList<E>, RandomAccess {
```

Comment:
The formatter is set to 120 characters in a line, I think it's fine to leave as is.

Reply:
If you were already modifying this, then sure. I don't think Ambarish's point was about the line being too long. In this case, the rewrapping is on otherwise unmodified lines of code, so is unrelated to your PR.

Reply:
Alright. I don't mind reverting, but I think that this form is easier to read than the multiline one.